### PR TITLE
Wins mobile margins

### LINKED ISSUE #1908: Remove additional space from wins-card-mobile.svg within WIns About page (mobile only)
### Overview
Remove the additional 31px of additional space from the `/assets/images/about/wins-card-mobile.svg`. We are currently using this `<svg>` within `pages/about/about-card-accomplishments.html`.

<details>
  <summary>Current About Page (mobile view)</summary>
  
</details>

<details>
  <summary>About Page (mobile view) figma design</summary>
  
</details>

<details>
  <summary>Current wins-card-mobile.svg</summary>
  
</details>

### Action Items
- [x] Research removing additional space from a `<svg>`.
- [x] update the `/assets/images/about/wins-card-mobile.svg` to remove the 31px of additonal space
- [x] verify the svg matches the figma design

### Resources/Instructions
- [/assets/images/about/wins-card-mobile.svg](https://github.com/hackforla/website/blob/gh-pages/assets/images/about/wins-card-mobile.svg)
- [My previous research from pull request 1870](https://github.com/hackforla/website/pull/1870#issuecomment-875169026)
- [SVG tutorial - review so you know what is going on in the svg](https://flaviocopes.com/svg/)
- [MDN - svg `viewBox` attribute](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/viewBox)
- [CSS Tricks - Using SVG](https://css-tricks.com/using-svg/)
- [remove padding from svg](https://stackoverflow.com/questions/14299571/removing-padding-from-svg)
- findings on comment down below

### PR DESCRIPTION
**Pull request by anonymousanemone**
Fixes #1908 

### What changes did you make and why did you make them ?

  - replaced wins-card-mobile.svg with a new svg that does not include shadows as part of the .svg
  - added shadows to wins-card-mobile.svg using css in _about.scss instead
  - added </a> about-card-accomplishments because wins-card-desktop had it, but mobile didn't

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals after changes are applied</summary>

after

</details>

<details>
<summary>Visuals before changes are applied</summary>
  
Screen Shot 2021-07-12 at 12 44 44 PM

</details>

### CODE REVIEW

File: _sass/components/_about.scss [hunk 1]
```diff
@@ -473,10 +473,9 @@ a.anchor {
 }
 
 .accomplishment-wins {
-  color: $color-red;
+  color: #333;
```
Comment (jbubar) [inline _sass/components/_about.scss]: could you use `$color-black` instead of `#333`. it is the same

**Review by jbubar — DISMISSED** [review]

File: _sass/components/_about.scss (hunk 1 above)
Comment (jbubar) [reply _sass/components/_about.scss]: ooh. actually, I think you can just delete that line, because that is the default color

Comment (anonymousanemone) [reply _sass/components/_about.scss]: fixed

Comment (anonymousanemone) [reply _sass/components/_about.scss]: @jbubar

**Review by Zak234 — DISMISSED** [review]
The changes you made look good and seem to be working. I'm pretty sure this isn't related to your issue but I noticed when I was checking your code changes through different phone resolutions that for a lot of the other phone resolutions besides iphone X that the wins-card is off-center. Should this be a new issue?

**Comment by Aveline-art** [comment]
@Zak234 Can you attach screenshots of what you are seeing vs the appearance of the mobile view on Figma?

File: pages/about/about-card-accomplishments.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
                 <p class="eliptical align-left coming-soon"><a href="https://hackforla.org/wins">View Volunteer Wins</a></p>
             </div>
             <div class="bottom-right-column wins-column-width-right">
-                <div class="wins-card-mobile"><a href="https://hackforla.org/wins" target="_blank" rel="noopener noreferrer"><img src="/assets/images/about/wins-card-mobile.svg" alt="The wins card displays the contributer's name, their teams and roles, and testimonials of their accomplishments." /></div>
+                <div class="wins-card-mobile"><a href="https://hackforla.org/wins" target="_blank" rel="noopener noreferrer"><img src="/assets/images/about/wins-card-mobile.svg" alt="The wins card displays the contributer's name, their teams and roles, and testimonials of their accomplishments." /></a></div>
```
Comment (adam-abundis) [inline pages/about/about-card-accomplishments.html:29]: Good catch. I kept trying to figure out why when I hovered over the `<a>` tag why it didn't cover the image. This is why.

**Review by adam-abundis — CHANGES_REQUESTED** [review]
Following up on @Zak234 comment. It looks great on desktop/mobile view. It seems the tablet view just aligns to the left.

<details>
  <summary>Click to see Tablet view (iPad) of About page (Accomplishments)</summary>
  
</details>

It looks like the the card flexbox converts to column in tablet view and since we are providing a width to the svg, it aligns to the left (start). So maybe add styling for tablet view (line 604-618).

**Comment by Zak234** [comment]
Screen Shot 2021-07-21 at 1 29 14 AM
Screen Shot 2021-07-21 at 1 29 57 AM
Screen Shot 2021-07-21 at 1 30 23 AM
Screen Shot 2021-07-21 at 2 23 46 AM

**Comment by anonymousanemone** [comment]
I've centered the image for "below tablet" which is when the screen is under 768px, but should I also center the image for when it is "bigger than tablet" but less than desktop? Here's how it looks:
Below tablet: 
image
Over tablet: 
image

**Comment by anonymousanemone** [comment]
Ask design team about whether to center or do something else

**Comment by kristine-eudey** [comment]
@anonymousanemone thanks for checking in about this. Please center the card for the "bigger than tablet" view as well.

**Comment by anonymousanemone** [comment]
I've made both of the changes requested above, please re-review! @jbubar @abuna1985

**Review by Zak234 — APPROVED** [review]
Looks good!

**Comment by macho-catt** [comment]
For the most part, changes on the page card itself looks good. Good job!

I also noticed that you've been working on this for a while, so props to you on that. However, I noticed one other thing that may not be in the jurisdiction of your issue/PR.

@Zak234 noticed it in one of his previous posts, but at a certain BP the parent cards are cutting off. This is due to the fact that flex (or any mobile responsive query) is not applied to the parent divs at specific BPs:
<details>
<summary>Click to see example</summary>

cut off below a certain bp

 </details>

The fix would be to apply flex (or some sort of media query) on the parent divs for each section of the About page.

@Aveline-art @abuna1985 is it better to open up a new issue for this? Otherwise, I would be inclined to approve this issue.

**Review by macho-catt — COMMENTED** [review]
For the most part, changes on the page card itself looks good. Good job!

I also noticed that you've been working on this for a while, so props to you on that. However, I noticed one other thing that may not be in the jurisdiction of your issue/PR.

@Zak234 noticed it in one of his previous posts, but at a certain BP the parent cards are cutting off. This is due to the fact that flex (or any mobile responsive query) is not applied to the parent divs at specific BPs:
<details>
<summary>Click to see example</summary>

cut off below a certain bp

 </details>

The fix would be to apply flex (or some sort of media query) on the parent divs for each section of the About page.

@Aveline-art @abuna1985 is it better to open up a new issue for this? Otherwise, I would be inclined to approve this issue.

**Comment by adam-abundis** [comment]
@macho-catt Go ahead and create a new issue for adding margin to the parent divs of the About page. Once that is complete, add it to this pr as a comment and approve the pull request. Then I will merge it. Thank you.

**Comment by macho-catt** [comment]
Opened a separate issue for the cutting off bug: #2077

**Review by macho-catt — APPROVED** [review]

**Pull request merged**